src/script/engines/lua.c

```diff
@@ -17,6 +19,8 @@
 
 #define MAX_KEY_SIZE 128
 
+mLOG_DECLARE_CATEGORY(SCRIPT);
```

Comment:
This is already done in `core/scripting.h`--though it should maybe be moved to `script/context.h` if you're using it here (there's a dependency graph between components, but I haven't actually explicitly defined it anywhere)

Reply:
Moved.

Reply:
You should remove it here then.